src/components/Landing/Landing.tsx

```diff
@@ -21,20 +21,37 @@ const Landing: React.FC<Props> = ({ title, subtitle }) => {
           />
         </figure>
        </div> */}
+
       <div className="media-content">
         <div className="card-header">
+          {/*
           <h1 className="card-header-title is-size-5 has-text-light">
             Audio for labels
-          </h1>
+          </h1>*/}
+        </div>
+
+        <div className="is-flex is-flex-direction-column is-justify-content-center is-align-items-center">
+          <div className="media">
+            <div className="block">
+              <i>Take or upload photo</i>
```

Comment:
hmm I don't think we should use the `<i>` tag. Can we access this in a css style instead?